dashboard/config/routes.rb

```diff
@@ -1139,10 +1139,11 @@
     end
 
     # Policy Compliance
-    get '/policy_compliance/child_account_consent/', to:
-      'policy_compliance#child_account_consent'
-    post '/policy_compliance/child_account_consent/', to:
-      'policy_compliance#child_account_consent_request'
+    namespace :policy_compliance do
+      get :pending_permission_request
+      get :child_account_consent
+      post :child_account_consent, action: :child_account_consent_request
+    end
```

Comment:
One option could be doing a sub-namespace like:

```
namespace :policy_compliance do
  namespace :child_accounts
    get :pending_permission_request
    get :consent
    post :consent, action: :consent_request
  end
  namespace :gdpr
    get :information_request
    post :deletion_request
  end
end
```

But since we don't actually seem to have any GDPR or other complaince-related routes, it might be YAGNI.